src/onepirate/modules/components/Snackbar.tsx

```diff
@@ -42,7 +39,7 @@ const styles = (theme: Theme) =>
     },
   });
 
-function Transition(props: TransitionProps) {
```

Comment:
Is there a strong type we can use here? Would prefer to avoid `any` if possible.

Reply:
I'm changing any with TransitionProps...
soon you will see the next commit.